Gaijinizer patch/Map471.txt

```diff
@@ -3,1663 +3,2153 @@
+> CONTEXT: Map471/events/1/pages/0/56/Dialogue/ruka_fc1
+Let's see...
 
+He's a demon that eats worlds, and whose natural
+form is similar to that of a black hole...?#Ron
 > END STRING
 
 > BEGIN STRING
 だが、\\C[2]本来は廃れた世界や滅んだ世界を『処理する』役割で
 
 無差別に世界を喰らうような存在ではない\\C[0]ということか。
-> CONTEXT: Map471/events/1/pages/0/61/Dialogue/enemy35 < UNTRANSLATED
-
+> CONTEXT: Map471/events/1/pages/0/61/Dialogue/enemy35
+But he was \\C[2]originally a being whose duty was to
+"tidy up" dead and dying worlds, not to
+indiscriminately devour any and all worlds.#Ron
 > END STRING
 
 > BEGIN STRING
 そうだね、合っているよ。
 
 今のアバドーンは正常ではない……
 いうなれば\\C[2]『暴走』\\C[0]してしまっている状態だ。
-> CONTEXT: Map471/events/1/pages/0/65/Dialogue/enemy27 < UNTRANSLATED
+> CONTEXT: Map471/events/1/pages/0/65/Dialogue/enemy27
+Yes, those are both right.
 
+Abaddon is not in his right mind any more. You
+could say he's \\C[2]completely out of control\\C[0] now.#Ron
 > END STRING
 
 > BEGIN STRING
 『暴走』…？
-> CONTEXT: Map471/events/1/pages/0/70/Dialogue/kazuya < UNTRANSLATED
-
+> CONTEXT: Map471/events/1/pages/0/70/Dialogue/kazuya
+Out of control?#Ron
 > END STRING
 
 > BEGIN STRING
 暴走が始まったきっかけは……そう
 
 \\C[2]あの『双子』の人形の身体を手に入れてからだ…\\C[0]
-> CONTEXT: Map471/events/1/pages/0/75/Dialogue/enemy27 < UNTRANSLATED
+> CONTEXT: Map471/events/1/pages/0/75/Dialogue/enemy27
+As for the cause of his rampage...
 
+\\C[2]It began after he acquired the doll bodies of
+those Twins.#Ron
 > END STRING
 
 > BEGIN STRING
 あれは、遙か未来のＭ.Ｏ.Ｗの生物兵器……
 
 \\C[10]『介入』\\C[2]の研究を進める過程の事故で生まれた個体……\\C[0]
-> CONTEXT: Map471/events/1/pages/0/79/Dialogue/enemy27 < UNTRANSLATED
-
+> CONTEXT: Map471/events/1/pages/0/79/Dialogue/enemy27
+They're a biological weapon created by M.O.W. in
+the distant future. \\C[2]They were born as a result of
+an accident in the process of \\C[10]Intervention
+\\C[0]research.#Ron
 > END STRING
 
 > BEGIN STRING
 か…\\C[10]『介入』\\C[0]の……！？
-> CONTEXT: Map471/events/1/pages/0/83/Dialogue/kazuya < UNTRANSLATED
-
+> CONTEXT: Map471/events/1/pages/0/83/Dialogue/kazuya
+I-\\C[10]Intervention\\C[0]?!#Ron
 > END STRING
 
 > BEGIN STRING
 \\N[1]くんの事象が起こる以前より
 あの組織は、『別次元からの干渉』に興味を持っていた。
 
 悪魔がそれを感覚的に知覚しているのを含めてね……
-> CONTEXT: Map471/events/1/pages/0/85/Dialogue/enemy27 < UNTRANSLATED
+> CONTEXT: Map471/events/1/pages/0/85/Dialogue/enemy27
+Before the events surrounding you happened,
+\\N[1]-kun, that organization had a keen
+interest in meddling with other dimensions.
 
+That included the ability of demons to perceive
+them.#Ron
 > END STRING
 
 > BEGIN STRING
 最終的に長い年月を経て、あの組織はその答えに辿り着いた
 
 けど結局、Ｍ.Ｏ.Ｗがアバドーンを従えることが出来ずに
 この\\C[2]『ロストナンバー』\\C[0]の計画は失敗に終わった……
-> CONTEXT: Map471/events/1/pages/0/90/Dialogue/enemy27 < UNTRANSLATED
-
+> CONTEXT: Map471/events/1/pages/0/90/Dialogue/enemy27
+And after a long time passed, they found the
+answer they were looking for. But without being
+able to subjugate Abaddon, the plans of this
+\\C[2]Lost Number\\C[0] would end in failure.#RonIffy
 > END STRING
```

Comment:
Minor revision for flow (and a tiny bit of localization:
```
After many years of research, they found the
answer they were looking for. But the \\C[2]Lost
Number\\C[0] project ended in failure because they
were unable to control Abaddon.
```